docs/characters.md

```diff
@@ -130,73 +129,66 @@ This table holds vital static information for each character. It is used to crea
 [38]: #trans_o
 [39]: #transguid
 [40]: #extra_flags
-[41]: #extra_flags
-[42]: #stable_slots
-[43]: #at_login
-[44]: #zone
-[45]: #death_expire_time
-[46]: #taxi_path
-[47]: #arenaPoints
-[48]: #totalhonorpoints
-[49]: #todayhonorpoints
-[50]: #yesterdayhonorpoints
-[51]: #totalkills
-[52]: #todaykills
-[53]: #yesterdayKills
-[54]: #chosenTitle
-[55]: #knownCurrencies
-[56]: #watchedFaction
-[57]: #drunk
-[58]: #health
-[59]: #power1
-[60]: #power2
-[61]: #power3
-[62]: #power4
-[63]: #power5
-[64]: #power6
-[65]: #power7
-[66]: #latency
-[67]: #talentGroupsCount
-[68]: #activeTalentGroup
-[69]: #exploredzones
-[70]: #equipmentcache
-[71]: #ammoid
-[72]: #knownTitles
-[73]: #actionbars
-[74]: #grantablelevels
+[41]: #stable_slots
+[42]: #at_login
+[43]: #zone
+[44]: #death_expire_time
+[45]: #taxi_path
```

Comment:
In the link ref you cannot use `_` write them as one word, all small letters. You do not need to change the names anywhere else. Only in this list.